tools/build/ci-jenkins-image/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM jenkins
+COPY bin/release-ci /usr/bin/release-ci
```

Comment:
Can we inline this? I feel like that's a little simpler to read

Reply:
sure, will make a follow up pr